src/packerlicious/community/builder.py

```diff
@@ -0,0 +1,146 @@
+"""
+Copyright 2018 Thor K. Hoegaas <thor at roht no>
```

Comment:
i think adding this will potentially complicate things for me later.

Reply:
I would hope not; the copyright notice does not change anything for your current license unless you want to change your license to an incompatible one at a later point?